src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -557,11 +566,16 @@ class CalculatorForm extends React.Component {
       (inputs.beneficiaryLocationQuestion === 'extension' &&
         inputs.extension === 'other')
     ) {
+      const errorMessage = this.state.invalidZip;
+
+      const errorMessageCheck =
+        errorMessage !== '' ? errorMessage : inputs.beneficiaryZIPError;
+
       amountInput = (
         <div>
           <ErrorableTextInput
-            errorMessage={inputs.beneficiaryZIPError}
-            label="Please enter the zip code where you’ll take your classes"
+            errorMessage={errorMessageCheck}
+            label="Please enter the Postal code where you'll take your classes"
```

Comment:
This needs to be zip code not Postal code.

Reply:
updated